src/jira/client/jira-client-audit-log-helper.ts

```diff
@@ -10,9 +10,8 @@ const getAuditInfo = ({
 	const auditInfo: Array<AuditInfo> = [];
 	const createdAt = new Date();
 	acceptedGithubEntities.map((githubEntityId) => {
-		// const repoEntities = repoData?.[githubEntityType];
-		const repoEntity = repoEntities.find(({ id }) => id === githubEntityId);
-		const issueKeys = repoEntity.issueKeys;
```

Comment:
Had to stringify the ID as the id is coming as string for branch and commit 
and as number for PR